### DISCUSSION TITLE
New action isn't showing

### DISCUSSION THREAD

**Discussion by Gethe** [community/community#26069] [Actions]
<p>I just committed a new <code>workflow_dispatch</code> action to my repo (<a href="https://github.com/Gethe/wow-ui-source" rel="nofollow noopener">https://github.com/Gethe/wow-ui-source</a>), but it doesn’t show in the actions list. I tested the exact action on a test private repo and it worked flawlessly there, so i’m not sure why it isn’t showing.</p>
<p>I’ve read that an action won’t show until it’s first run, but since this is a <code>workflow_dispatch</code> action I need to see it in the actions tab first to run it. catch 22</p>

**Comment by RammusXu** [comment]
<p>How did you trigger the <code>workflow_dispatch</code>?</p>
<p>Did you set the right <code>workflow_id</code>? like <a href="https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event" rel="nofollow noopener">https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event</a></p>

**Comment by Gethe** [comment]
<p>In the test repo i triggered it from the button on the actions tab, but i don’t have that option with the public repo.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/gethe">@Gethe</a>,</p>
<p>Glad to see you in Github Community Forum!</p>
<p>Please comment/delete the current trigger and add <code>on: workflow_dispatch</code> again.  I forked your repository and it works on my side.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e78ff9fa6af3e3f5a6d3c2b60f538051ce9c3001.png" data-download-href="/uploads/short-url/x2uLmfJIOzZJVNAxPhP5NzDW4sF.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181095066-3750d20c-0a71-46e2-bf3a-0b64abeaa73e.png" alt="image" data-base62-sha1="x2uLmfJIOzZJVNAxPhP5NzDW4sF" width="690" height="203" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e78ff9fa6af3e3f5a6d3c2b60f538051ce9c3001_2_690x203.png, https://aws1.discourse-cdn.com/github/optimized/2X/e/e78ff9fa6af3e3f5a6d3c2b60f538051ce9c3001_2_1035x304.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/e/e78ff9fa6af3e3f5a6d3c2b60f538051ce9c3001_2_1380x406.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e78ff9fa6af3e3f5a6d3c2b60f538051ce9c3001_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1435×423 39.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thanks</p>

**Reply by abhinavan1812** [reply]
Hi @weide-zhou,
I have tried this in my workflow, but didn't work for me.

**Comment by Gethe** [comment]
<p>Oh cool, that fixed it. Thanks!</p>
